bindata/bootkube/scc-manifests/0000_20_kube-apiserver-operator_00_scc-anyuid.yaml

```diff
@@ -6,22 +17,11 @@ allowHostPorts: false
 allowPrivilegeEscalation: true
 allowPrivilegedContainer: false
 allowedCapabilities:
-apiVersion: security.openshift.io/v1
 defaultAddCapabilities:
 fsGroup:
   type: RunAsAny
 groups:
 - system:cluster-admins
-kind: SecurityContextConstraints
```

Comment:
this just moving?  if so, moving all "just moves" into a separate PR makes it easier to approve

Reply:
Indeed, as requested by @s-urbaniak. I can move it to a different PR :+1: